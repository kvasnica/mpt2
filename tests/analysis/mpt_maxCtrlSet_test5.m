function mpt_maxCtrlSet_test5

% tests whether mpt_maxCtrlSet() works with Options.Vconverge>0 and
% Options.Kinf=0:
% http://control.ee.ethz.ch/~mpt/hg/mpt?cs=c93bd71dd917

lti1d
Options.Vconverge = 30;
[P, iter] = mpt_maxCtrlSet(sysStruct, Options); % Options.Kinf=1 should be set automatically
mbg_assertequal(iter, 3);  % 3 iterations should have been needed with Options.Vconverge=30
