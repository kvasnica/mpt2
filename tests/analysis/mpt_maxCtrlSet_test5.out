Computing Kinf set because Options.Vconverge>0...
Iteration Number 1
Iteration 1       
Volume difference (10.56 %) below tolerance (30.00 %), aborting computation...
